soc/nxp/imx/imx9/Kconfig.defconfig

```diff
@@ -1,8 +1,6 @@
-# Copyright 2022,2024 NXP
+# Copyright 2024 NXP
```

Comment:
Don't remove 2022?

Reply:
I moved files under imx9 to imx9/imx93. So, this is a actually a new file I write manually.

Reply:
okay, thanks.